export/export_pipeline/compute_gltf_representation.py

```diff
@@ -0,0 +1,270 @@
+"""
+Run this script from UE
+Even though we eventually would like to run physics in a left-handed coordinate system to match UE, this script
+exports everything in the right-handed coordinate system. This is done to maintain script-level consistency, because
+GLTF export (implemented by UE) always exports objects in a right-handed coordinate system. The unit of translation 
+can be set, however. It affects both the offsets and the geometry. So we set that to cm, overriding the default
+GLTF exporter behaviour of expressing translation in m. Unit of rotation (joint limits and references) is degrees.
+"""
+
+import argparse
+from collections import namedtuple
+import json
+import os
+import time
+import unreal
+import yaml
+
+
+osp = os.path
+ss = unreal.get_editor_subsystem(unreal.UnrealEditorSubsystem)
+world = ss.get_editor_world()
+
+
+TRANS_UE_TO_MUJOCO_SCALE = 1.0  # keep cm
+ROT_UE_TO_MUJOCO_SCALE = 1.0  # keep degrees
+
+
+def export_meshes(actor, export_dir):
+  unreal.EditorLevelLibrary.set_selected_level_actors([actor, ])
+  name = actor.get_actor_label()
+  filename = osp.join(export_dir, f'{name}.gltf')
+  task = unreal.AssetExportTask()
+  task.set_editor_property('selected', True)
+  task.set_editor_property('automated', True)
+  task.set_editor_property('prompt', False)
+  task.set_editor_property('object', world)
+  task.set_editor_property('filename', filename)
+  # unreal.GLTFExporter.set_editor_property('export_task', task)
+  options = unreal.GLTFExportOptions()
+  options.set_editor_property('export_cameras', False)
+  options.set_editor_property('export_lights', False)
+  options.set_editor_property('export_uniform_scale', TRANS_UE_TO_MUJOCO_SCALE)
+  options.set_editor_property('export_unlit_materials', False)
+  options.set_editor_property('export_vertex_colors', False)
+  selected_actors = unreal.Set(unreal.Actor)
+  selected_actors.add(actor)
+  print(f'Writing actor {name} to {filename}...')
+  count = 0
+  N = 10
+  while not unreal.GLTFExporter.export_to_gltf(None, filename, options, selected_actors):
```

Comment:
- Why do we need this retry logic?
- A for loop would be cleaner.
- Assert rather than pass around error codes.

Reply:
we don't need the retry logic, removed it and asserted instead.